nixos/modules/installer/cd-dvd/iso-image.nix

```
@@ -548,7 +574,20 @@ in
       '';
       description = lib.mdDoc ''
         This option lists files to be copied to fixed locations in the
-        generated ISO image.
+        main data partition of the generated ISO image.
+      '';
+    };
+
+    isoImage.bootContents = mkOption {
+      example = literalExpression ''
+        [ { source = ./your-custom-efi-app.efi;
+            target = "EFI/boot/your-custom-efi-app.efi";
+          }
```

```suggestion
        } ]
```
Let's keep that compact